index.html

```diff
@@ -698,7 +698,7 @@ <h4>
         </li>
       </ul>
       <div class="note" id="n005">
-        <p its-locale-filter-list="en" lang="en" class="checkme">Establishing a type area may be seen as defining not only a rectangular area on a page, but also within that area, an underlying, logical grid to guide the placement of such things as characters, headings, and illustrations. Once the grid is established according to the principles of composition, the setting of the characters must align with the grid. If the content contains Chinese text only, it is important that the first and last characters on a line should align with the border of the type area. When both Chinese text and Western text are mixed in the content, or forbidden locations of punctuation marks need to be taken into consideration, the setting of the content may not align with the grid.</p>
+        <p its-locale-filter-list="en" lang="en" class="checkme">Establishing a type area may be seen as defining not only a rectangular area on a page, but also within that area, an underlying, logical grid to guide the placement of such things as characters, headings, and illustrations. Once the grid is established according to the principles of composition, the setting of the characters must align with the grid. If the content contains Chinese characters only, it is important that the first and last characters on a line should align with the border of the type area. When both Chinese characters and European text are mixed in the content, or forbidden locations of punctuation marks need to be taken into consideration, the setting of the content may not align with the grid.</p>
```

Comment:
For the purposes of these edits, are fullwidth punctuation and letters based on ASCII to be considered as Chinese characters?  Perhaps we need to define 'Chinese character' and 'European character' using character lists in the glossary/appendix, and then point to those definitions when we use the term ??  Or... perhaps the distinction we're trying to make here is between fullwidth and non-fullwidth characters, rather than Chinese vs European? (That would make it easier to fit text in other scripts etc, into what we're saying...)

Other terms that may be appropriate substitutes for non-fullwidth could be 'narrow' or 'proportionally-spaced', depending on the context.

Reply:
There seem to be some conflicts between this PR and [#504](https://github.com/w3c/clreq/pull/504/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R701).